apps/etterlatte-samordning-vedtak/src/main/kotlin/no/nav/etterlatte/samordning/vedtak/SamordningVedtakRoute.kt

```diff
@@ -110,6 +110,30 @@ fun Route.samordningVedtakRoute(
             call.respond(samordningVedtakDtos)
         }
 
+        get("/har-loepende-oms") {
+            val paaDato =
+                call.parameters["paaDato"]?.let { runCatching { LocalDate.parse(it) }.getOrNull() }
```

Comment:
Hadde kanskje vore fint med ein extension function på call som parser til dato, trur det er ein sånn ting vi gjer på fleire forskjellige måtar, så å leggje til rette for å gjera det likt kunne gjort seg.

Reply:
Lagt til det nå